### PR TITLE
Project Database model

### LINKED ISSUE #25: Project Database Model
Create a model in the database for projects. This should contain all relevant data associated with a project and be associated with a user.

### PR DESCRIPTION
**Pull request by mxchen2001**
### Problem
We had no database model to contain the information about the projects that users interact with.

### Solution
Create a class using Mongoengine to contain hardware set data. Currently, it contains the name, descriptions, and user id of the owner.

### Testing
I created a test route where I populated dummy data into MongoDB to make sure the entry is valid

Closes #25

### CODE REVIEW

File: server/api/routes/projects.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    return
+@projects.route('/update_projects')
+def update_projects():
+
+    return
+@projects.route('/delete_projects')
+def delete_projects():
+
+    return
```
Comment (JimothyGreene) [inline server/api/routes/projects.py]: All of these routes aren't part of this PR. Should separate it so these are created when you're handling #27

**Review by JimothyGreene — CHANGES_REQUESTED** [review]
Model looks good. I would just uninclude the addition of routes/projects.py for this PR since that has its own issue.

**Review by ishan0102 — CHANGES_REQUESTED** [review]
Same comment as @JimothyGreene about splitting the routes into a separate PR.

**Review by mxchen2001 — COMMENTED** [review]
Removed the routes in `routes/projects.py`

**Review by mxchen2001 — COMMENTED** [review]
Removed blueprints in  `__init__.py`

**Review by JimothyGreene — COMMENTED** [review]
Looks good. You'll just have to merge `main` into this branch and resolve the merge conflicts. And then just make sure what you added still works.

**Review by JimothyGreene — APPROVED** [review]
LGTM

**Review by ishan0102 — APPROVED** [review]
Lgtm

**Pull request merged**